tests/test_utils.py

```diff
@@ -270,17 +270,17 @@ def _test_function2(out_dir: Path, foo: bool = False, bar: int = 1):
 @pytest.mark.parametrize(
     "command",
     [
-        "any.py",
+        #"any.py",
```

Comment:
```suggestion
        "any.py",
```
could you check if it works now?